### PR TITLE
[New Feature] Google maps extra options 

### PR DESCRIPTION
**Pull request by jonnyforestGIS**
# Description

This PR will give extra configuration in Google Maps. You can set the Level of Zoom and the type of base map that you want to see in the location of the supplier and locations.

Documentation Support Google Maps:
https://developers.google.com/maps/documentation/maps-static/start

Fixes # (issue)
NA

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Test extra parameters adding in env file:
GOOGLE_MAPS_API=<Insert your key>
GOOGLE_MAPS_MAP_TYPE_BASEMAP=hybrid 
GOOGLE_MAPS_ZOOM_LEVEL=21

With these variables, the result when adding a new location or supplier is to create a location image with satellite imagery and the labels of the streets and POIs.

**Test Configuration**:
* PHP version: 7.4.3
* MySQL version: 11.2.3
* Webserver version: Apache 2.4
* OS version: Windows 10 Pro

# Checklist:

- [x] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [x] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

#Observations:

Is my first PR in this project, I'm sorry if I miss some guideline of your contribution code.
Greetings from Portugal.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -130,6 +130,14 @@
         "description": "Include your Google Maps API key here if you'd like Snipe-IT to load maps from Google on your locations and suppliers pages.",
         "required": false
       },
+      "GOOGLE_MAPS_MAP_TYPE_BASEMAP": {
```
Comment (snipe) [inline app.json:133]: Is this file generated? I don't even know what this is?

File: .env.example [hunk 2]
```diff
@@ -177,6 +176,14 @@ REQUIRE_SAML=false
 API_THROTTLE_PER_MINUTE=120
 CSV_ESCAPE_FORMULAS=true
 
+# --------------------------------------------
+# OPTIONAL: GOOGLE MAPS SETTINGS
+# --------------------------------------------
+GOOGLE_MAPS_API=
```
Comment (snipe) [inline .env.example:182]: Let's set some sane defaults here, or at least set them to null

File: config/services.php [hunk 3]
```diff
@@ -52,6 +52,8 @@
 
     'google' => [
         'maps_api_key' => env('GOOGLE_MAPS_API'),
+        'maps_type_basemap' => env('GOOGLE_MAPS_TYPE_BASEMAP'),
```
Comment (snipe) [inline config/services.php:55]: Please use sane defaults here as well - not everyone will go back in and update the .env file to have a value, and we don't want it to be nothing at all if the value is null or missing

**Review by snipe — CHANGES_REQUESTED** [review]
Thanks! Some changes requested.

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this PR for now

**Comment by jonnyforestGIS** [comment]
Hi @snipe sorry for my delay. 
Meanwhile my daughter born and I wasn't enable to answer or change your comments.   
I'm sorry 😔 When I have more time I return to this.

**Pull request closed without merging**